bugzilla2gitlab/models.py

```diff
@@ -246,13 +252,13 @@ class Attachment(object):
     '''
     The attachment model
     '''
-    def __init__(self, bugzilla_attachment_id, filename):
+    def __init__(self, bugzilla_attachment_id, file_description):
         self.id = bugzilla_attachment_id
-        self.filename = filename
+        self.file_description = file_description
         self.headers = conf.default_headers
 
     @classmethod
-    def parse_filename(cls, comment):
+    def parse_file_description(cls, comment):
```

Comment:
I think this rename is file, because you are parsing a file description to extract the name.

Reply:
I think it's fine too, I've tried to explain this above. Sorry for the original code being unclear.